primitives/teerex/src/lib.rs

```diff
@@ -48,9 +47,6 @@ impl<PubKey, Url> Enclave<PubKey, Url> {
 
 pub type ShardIdentifier = H256;
 
-// Disambiguate associated types
-pub type AccountId<T> = <T as frame_system::Config>::AccountId;
-
```

Comment:
maybe you don't need the frame-system dependency anymore.

Reply:
Still needed I think

Reply:
I am quite sure it isn't.

Reply:
you're right